### DISCUSSION TITLE
Problem with homebrew installation

### DISCUSSION THREAD

**Discussion by dectun** [Homebrew/discussions#3188] [Getting started]
Hello. I try instal homebrew in a mac M1. 

Then the installation it say me; "Warning: /opt/homebrew/bin is not in your PATH.
  Instructions on how to configure your shell for Homebrew
  can be found in the 'Next steps' section below."
  
Then I try whit "brew doctor" and it say me; "zsh: command not found: brew"

Then I try export de variables, like this link;
(https://github.com/simonfuhrmann/mve/wiki/Build-Instructions-for-OS-X)

with these commands;
![Captura de Pantalla 2022-04-16 a las 3 06 42](https://user-images.githubusercontent.com/103385292/163655933-5e170471-a737-4282-a0d7-2c9020350dc9.png)

Here I put the screenshot of the terminal;
![Captura de Pantalla 2022-04-16 a las 3 03 13](https://user-images.githubusercontent.com/103385292/163655955-b8e0becb-5d23-4001-bb8f-cb0736135edc.png)

**Comment by dectun** [comment]
Then I try with the instruction of the promt, that the installation dive me. And it say; "zsh: permission denied: /Users/dectun/.zprofile"
![Captura de Pantalla 2022-04-16 a las 3 13 16](https://user-images.githubusercontent.com/103385292/163656171-47d490ee-7da5-41c3-b6a1-62cf601c6728.png)

**Reply by gromgit** [reply]
First, please don't post screenshots of text output. They're really hard to read on mobile device, aren't searchable, and you're forcing people to manually retype long strings just to help you. **Always post text as text**.

Now to the real problem. The *Next steps* clearly states the two commands you need to run as:
```
echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/dectun/.zprofile
eval "$(/opt/homebrew/bin/brew shellenv)"
```
but you ran this instead:
```
echo 'eval "$(/opt/homebrew/bin/brew shellenv)"'
/Users/dectun/.zprofile
eval "$(/opt/homebrew/bin/brew shellenv)"
```
See the difference?

**Reply by dectun** [reply]
Ok, very much thanks. It´s work.

And thanks for the tip that you give me. I will not use the screenshot. So sorry.
